lnrpc/signrpc/signer_server.go

```diff
@@ -547,3 +567,34 @@ func (s *Server) DeriveSharedKey(_ context.Context, in *SharedKeyRequest) (
 
 	return &SharedKeyResponse{SharedKey: sharedKeyHash[:]}, nil
 }
+
+// parseRawKeyBytes checks that the provided raw public key is valid and returns
+// the public key.
+func parseRawKeyBytes(rawKeyBytes []byte) (*btcec.PublicKey, error) {
+	var (
+		targetPubKey *btcec.PublicKey
+		err          error
+	)
+
+	switch {
+	// If the user provided a raw key, but it's of the
+	// wrong length, then we'll return with an error.
+	case len(rawKeyBytes) != 0 && len(rawKeyBytes) != 33:
+
```

Comment:
nit (pre-existing): can remove extra newline as the condition isn't multi-line.

Reply:
fixed